CONTRIBUTING.md

```diff
@@ -16,28 +16,26 @@ which details these points and more, and is a recommended read.
 
 ## Reporting bugs
 
-Report bugs [here](https://github.com/redotengine/rdot/issues/new?assignees=&labels=&template=bug_report.yml).
+Report bugs [here](https://github.com/Redot-Engine/redot-engine/issues/new?assignees=&labels=&template=bug_report.yml).
 Please follow the instructions in the template when you do.
 
 Notably, please include a Minimal Reproduction Project (MRP), which is a small
-Godot project which reproduces the issue, with no unnecessary files included.
+Redot project which reproduces the issue, with no unnecessary files included.
 Be sure to not include the `.godot` folder in the archive to save space.
 
-Make sure that the bug you are experiencing is reproducible in the latest Godot
-releases. You can find an overview of all Godot releases [on the website](https://godotengine.org/download/archive/)
-to confirm whether your current version is the latest one. It's worth testing
-against both the latest stable release and the latest dev snapshot for the next
-Godot release.
+Make sure that the bug you are experiencing is reproducible in the latest Redot
+releases. It's worth testing against both the latest stable release and the latest dev snapshot for the next
```

Comment:
Need reformatting

Reply:
Need more detailed description than that. It looks good on my local repo and its the same text as the Godot original, just with a sentence removed